playbooks/common/openshift-docker/config.yml

```diff
@@ -0,0 +1,85 @@
+- name: Set facts necessary to configure docker
+  hosts:
+    - masters
+    - nodes
```

Comment:
These groups are not generally available in playbooks/common, they are only used in the byo playbooks...

evaluate groups sets up the following groups:
oo_masters_to_config
oo_nodes_to_config
oo_etcd_to_config
oo_lb_to_config
oo_first_master
oo_first_etcd

We should use these groups (or newly created groups for common playbooks)